Cargo.lock

```diff
@@ -2921,18 +2943,18 @@ checksum = "ff011a302c396a5197692431fc1948019154afc178baf7d8e37367442a4601cf"
 
 [[package]]
 name = "openssl-src"
-version = "300.1.6+3.1.4"
```

Comment:
The build error https://github.com/confidential-containers/kbs/actions/runs/7483975835/job/20370068894?pr=283#step:7:849 is caused by chaing the `version` and `checksum` here. Although `Cargo` will automatically update the rev here, but we'd better keep it as-is to make the csv building pass.

At the meantime, we are getting touch with csv colleagues to fix the upstream.

Reply:
Thanks for the tip!